src/index.js

```diff
@@ -1,11 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
-const App = () => (
-  <div>
-    <h1>Calculator App</h1>
-  </div>
-);
+import App from './components/App';
 
 ReactDOM.render(
   <App />,
```

Comment:
The `src/index.js` should:

- [ ] add example books to initial state object <a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png">screenshot from pathwright</a>
- [ ] import { Provider } from react-redux